### ISSUE TITLE
How to get loggedin User oauth token and oauth secret key for linkedin?

### ISSUE THREAD

**Issue by Surendra-04** [heresy/angularjs-social-login#28]
HI ,,
How to get loggedin User oauth token and oauth secret key for linkedin? Please help me to get both values.

**Existing code for linkedin**

IN.User.authorize(function(token){
                    console.log(token);
                                   
					IN.API.Raw("/people/~:(id,first-name,last-name,email-address,picture-url)").result(function(res){
						socialLoginService.setProvider("linkedIn");
						var userDetails = {name: res.firstName + " " + res.lastName, email: res.emailAddress, uid: res.id, provider: "linkedIN", imageUrl: res.pictureUrl};
						$rootScope.$broadcast('event:social-sign-in-success', userDetails);
				    });
				});

**Comment by sreejithcj** [comment]
Yes. Even I need Access token from both LinkedIN and Facebook. How do I get that?

**Comment by ruisilva450** [comment]
Any news on this? I need this for google login
